tfe/resource_tfe_organization_token_test.go

```diff
@@ -99,6 +99,52 @@ func TestAccTFEOrganizationToken_existsWithForce(t *testing.T) {
 	})
 }
 
+func TestAccTFEOrganizationToken_existsWithoutExpiry(t *testing.T) {
+	token := &tfe.OrganizationToken{}
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+	orgName := fmt.Sprintf("tst-terraform-%d", rInt)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckTFEOrganizationTokenDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTFEOrganizationToken_existsWithoutExpiry(rInt),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Since we have a test here for checking that tokens can be created without an expiry, is it possible to add a check that the created token has a null expiration date? This would help guard against regressions where we dont return properly [on this line](https://github.com/hashicorp/terraform-provider-tfe/pull/844/files#diff-3ba6ff1c80cd647a2c785e0c5f5fb72a8aa985181fb2689b41686131d98bfb0fR92), for example.

The same would also be nice on the team tokens, if possible.

Reply:
Done! ✅